app/uk/gov/hmrc/essttpstubs/model/EligibilityRequest.scala

```diff
@@ -16,12 +16,12 @@
 
 package uk.gov.hmrc.essttpstubs.model
 
+import essttp.rootmodel.ttp.eligibility.Identification
 import play.api.libs.json.{Format, Json}
 
 final case class EligibilityRequest(
     channelIdentifier:         String,
-    idType:                    String,
-    idValue:                   String,
+    identification:            List[Identification],
```

Comment:
we still need the stub to read the old version of the request as well in case we want to test against the old world still - we'd just need a writes instance of this request which is able to handle both versions